content/components/action-list.mdx

```diff
@@ -224,13 +224,13 @@ Action menus are used for disambiguation, navigation, or to display secondary op
 
 Dropdown menus are used for making a single selection among a small list of options. The list is usually predefined with system values, but in some cases it can include user-provided data when those are known not to grow into too many items.
 
-[Primer React implementation](https://primer.style/components/DropdownMenu)
+[Primer React implementation](/react/DropdownMenu)
```

Comment:
This looks like it's going to /design/react/DropdownMenu in the deployment preview. @maximedegreve can you double check that all image and text links here are doing what they should?

Reply:
I've switched them back to absolute links as I'm pretty sure it won't work because the router isn't the same between the interface site and the react site. 😅 This risks making mistakes as we can't test it on the deploy previews. Check it now @yaili  it should work now I think. 👀